cmd/lsif-go/main.go

```diff
@@ -0,0 +1,100 @@
+// The program lsif-go is an LSIF exporter for Go.
```

Comment:
Should we change this "exporter" to "indexer"?

Reply:
Will change all of these in another PR.